src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -363,6 +362,12 @@ export class QuickBar extends LitElement {
 
         mwc-list-item {
           width: 100%;
+          --mdc-ripple-color: transparent;
```

Comment:
Why not set the ripple color? Would fix your issue

Reply:
Haha, to be honest I was just pasting from the docs til something worked :P I'll do that

Reply:
Hm. That doesn't fix the "first item not highlighting on launch" issue I don't think. This changes color for all list items, but we need the first item to receive focus once the list renders for it to trigger the `:focus` stylings.

Reply:
It doesn't use the focus style, we trigger the ripple in `this_setFocusFirstListItem`